doc/rm/comportability_specification/index.md

```diff
@@ -440,18 +440,70 @@ Unlike interrupts, there is no software component to alerts at the peripheral, t
 See that [specification]({{< relref "/hw/ip/alert_handler/doc" >}}) for full details.
 A general description of the handling of alerts at the hardware level is given here.
 
-### Alerts per module
+### Alerts per Module
 
 Alerts are sent as a bundled output from a peripheral to the hardware alert handler.
 Each peripheral can send zero or more alerts, where each is a distinguishable security threat.
 Each alert originates in some internal event, and must be specially handled within the peripheral, and then within the alert handler module.
 
+Alerts of comportable IPs in the system must be in either of the following two categories:
+
+1. *Recoverable*, one-time triggered alerts.
+This category is for regular alerts that are due to recoverable error conditions.
+The alert sender transmits one single alert event when the corresponding error condition is asserted.
+
+2. *Fatal* alerts that are continuously triggered until reset.
+This category is for highly critical alerts that are due to terminal error conditions.
+The alert sender continuously transmits alert events until the system is reset.
+
+It is recommended that fatal alerts also trigger local security countermeasures, if they exist.
+For example, a redundantly encoded FSM that is glitched into an invalid state is typically considered to be a fatal error condition.
+In this case, a local countermeasure could be to move the FSM into a terminal error state in order to render the FSM inoperable until the next reset.
+
+The table below lists a few common error conditions and the recommended alert type for each of those errors.
+
+Error Event                                                | Regular IRQ | Recoverable Alert | Fatal Alert
+-----------------------------------------------------------|-------------|-------------------|-------------
+ECC correctable in NVM (OTP, Flash)                        | x           | x                 |
+ECC uncorrectable in Flash                                 | x           | x                 |
+ECC uncorrectable in OTP                                   | (x)         |                   | x
+Any ECC / Parity error in SRAMs or register files          | (x)         |                   | x
+Glitch detectors (e.g., invalid FSM encoding)              | (x)         |                   | x
+Incorrect usage of security IP (e.g., AES config register) | x           | x                 |
```

Comment:
Should the signaling as regular IRQ also be optional for security IP? At least in AES, there are no interrupts. You may want to write "shadowed control register in AES" instead of "AES config register".

Reply:
oh yeah let me make that optional.

I'll update the text to include `shadowed control register in AES`.

Reply:
Updated.

Reply:
Thanks Michael.